security/src/main/java/uk/ac/ox/ndph/mts/security/authorisation/AuthorisationService.java

```diff
@@ -100,7 +101,8 @@ public boolean authorise(String requiredPermission, List<String> entitiesSiteIds
         try {
             //Get the user's object id
             String userId = securityContextUtil.getUserId();
-            String token = securityContextUtil.getToken();
+            String tokenString = securityContextUtil.getToken();
+            Consumer<org.springframework.http.HttpHeaders> token = PractitionerServiceClient.bearerAuth(tokenString);
```

Comment:
Not sure, but isn't token name a bit misleading? Shouldn't be something like authHeaders?

Reply:
I agree, fixed